### ISSUE TITLE
Make RemoveFilesFromZipTask vastly faster

### ISSUE THREAD

**Issue by stefan-reich** [srikanth-lingala/zip4j#405]
I didn't exactly make a pull request, but I have a very similar class showing how to do it ([Zip4j_CopySubSetOfZipFile] (https://code.botcompany.de:9898/tb-int/get-transpiled.php?id=1034356&raw=1)).

Example measurement: For a 9 MB zip file with 2k entries of which I keep about 1k entries totalling 5 MB, my version reduces the runtime from ~480 ms to ~180 ms.

**Comment by srikanth-lingala** [comment]
Hi, thanks for pointing out this improvement. To be honest, the code in your link is relatively large. I need some time to go through it, and incorporate those changes in zip4j. I will not be able to get it into the next release. I will consider this for the release after the next.

**Comment by stefan-reich** [comment]
Yeah I get it... plus my coding style is probably different from yours. No worries!

**Comment by srikanth-lingala** [comment]
As much as I would love to have such an improvement in zip4j, I don't have enough time to understand the code in the link you provided. I really appreciate your effort in improving zip4j, but the code in your link will take considerable time to understand, which I am really extremely short on. I will close this issue, but will definitely consider this if I find enough time at some point.

Thanks again for your contribution.
